app/src/main/java/net/kuama/documentscanner/utils/PerspectiveTransformUtils.kt

```diff
@@ -0,0 +1,36 @@
+package net.kuama.documentscanner.utils
+
+import org.opencv.core.Point
+import java.util.*
+import kotlin.Comparator
+
+object PerspectiveTransformUtils {
+    fun sortPoints(sourcePointsArray: Array<Point>): Array<Point> {
```

Comment:
wouldn't be better to have a square or similar object as parameter? it can be an array with any number of fields